framework/simd.conf

```diff
@@ -90,11 +90,11 @@ cmpccxadd		Leaf07_01EAX	    6		# CMPccXADD instructions
 #fred			Leaf07_01EAX	    17		# Flexible Return and Event Delivery
 #lkgs			Leaf07_01EAX	    18		# Load into Kernel GS
 amxfp16			Leaf07_01EAX	    21	amx	# AMX Tile multiplication in FP16
-#avxifma		Leaf07_01EAX	    23		# AVX-IFMA instructions
+avxifma			Leaf07_01EAX	    23		# AVX-IFMA instructions
 #lam			Leaf07_01EAX	    26		# Linear Address Masking
-#avxvnniint8		Leaf07_01EDX	    4	avx	# AVX Vector Neural Network Instructions, Int8
-#avxneconvert		Leaf07_01EDX	    5	avx	# AVX Non-Exception BF16/FP16/FP32 Conversion instructions
-#prefetchiti		Leaf07_01EDX	    14		# PREFETCHIT0/1 instructions
+avxvnniint8		Leaf07_01EDX	    4	avx	# AVX Vector Neural Network Instructions, Int8
+avxneconvert		Leaf07_01EDX	    5	avx	# AVX Non-Exception BF16/FP16/FP32 Conversion instructions
+prefetchiti		Leaf07_01EDX	    14		# PREFETCHIT0/1 instructions
```

Comment:
This shouldn't be necessary, as the instructions execute as no-op when the feature isn't present. Tests shouldn't perform a CPUID check for this.